Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -3,13 +3,18 @@
 
 import {Artifact, Cmd, Transformer, Tool} from "Sdk.Transformers";
 
-const root = Environment.hasVariable("QTEST_DEPLOYMENT_PATH") ? d`${Environment.getFileValue("QTEST_DEPLOYMENT_PATH")}` : d`.`;
-const qCodeCoverageEnumType = Environment.hasVariable("[Sdk.BuildXL]qCodeCoverageEnumType")
-    ? Environment.getStringValue("[Sdk.BuildXL]qCodeCoverageEnumType")
-    : "None";
+const root = d`.`;
 
-const isChangeBasedCodeCoverage = Environment.getFlag("[Sdk.BuildXL]inputChangesPresented");
+const codeCoverageOption = Environment.hasVariable("[Sdk.BuildXL.CBInternal]CodeCoverageOption")
+    ? Environment.getStringValue("[Sdk.BuildXL.CBInternal]CodeCoverageOption")

```

Comment:
>Environment.getStringValue("[Sdk.BuildXL.CBInternal]CodeCoverageOption") [](start = 6, length = 72)

Should this be 
```
Environment.getStringValue("[Sdk.BuildXL.CBInternal]CodeCoverageOption") || CoverageOptions.None.toString();
```
?

Reply:
Another nice thing is to have a `switch` for string of `[Sdk.BuildXL.CBInternal]CodeCoverageOption`. Then the value of `codeCoverageOption` should be of type `CoverageOptions`. 

---
In reply to: [332765394](https://github.com/microsoft/BuildXL/pull/996#discussion_r332765394) [](ancestors = 332765394)